apps/coscrad-frontend-e2e/src/e2e/tags/create-tag.e2e.cy.ts

```diff
@@ -0,0 +1,35 @@
+const commandLabel = `Create Tag`;
+
+describe(`Create Tag`, () => {
+    beforeEach(() => {
+        cy.visit('/');
+
+        cy.clearDatabase();
+
+        cy.executeCommandStream('users:create-admin');
```

Comment:
Sweet!!

Reply:
What's interesting about this is we could have a separate command stream that registers the cypress user but adds the regular viewer role or no role at all. Then we can test properly that the commands only show up for admin users.